opennms-container/minion/container-fs/confd/templates/prom-jmx-exporter.yaml.tmpl

```diff
@@ -0,0 +1,29 @@
+{{if exists "/java/agent/prom-jmx-exporter" -}}
+#
+# DON'T EDIT THIS FILE :: GENERATED WITH CONFD
+#
+
+{{$jmxExporterPath := "/java/agent/prom-jmx-exporter" -}}
+jmxUrl: '{{getv (print $jmxExporterPath "/jmxUrl") "service:jmx:rmi:///jndi/rmi://127.0.0.1:1299/karaf-minion"}}'
+username: {{getv (print $jmxExporterPath "/username") "admin"}}
+password: {{getv (print $jmxExporterPath "/password") "admin"}}
+lowercaseOutputName: {{getv (print $jmxExporterPath "/lowercaseOutputName") "true"}}
+lowercaseOutputLabelNames: {{getv (print $jmxExporterPath "/lowercaseOutputLabelNames") "true"}}
+whitelistObjectNames: {{getv (print $jmxExporterPath "/whitelistObjectNames") `[ "org.opennms.core.ipc.sink.producer:*", "org.opennms.netmgt.dnsresolver.netty:*", "org.opennms.netmgt.telemetry:*" ]`}}
```

Comment:
Could you move the `[]` directly into the template, omitting them from the default value specified here? If we do that, we can use the `set` type in the schema rather than `string` which will render the config form a bit nicer.

See the `bootstrap-servers` for an example where we are using a `set` and concating together with `,` which we could do here.

In the future we can specify validation for the elements to ensure they are quoted in the correct form, but that can wait.